lessons/pydata/pandas_groupby/index.ipynb

```diff
@@ -15,7 +15,7 @@
   },
```

Comment:
<p>Heh, to jsem nevěděl. #TIL</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/pyladies-kurz/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>